changelogs/fragments/nmap_dns_servers.yml

```
@@ -0,0 +1,3 @@
+---
+minor_changes:
+  - community.general.nmap - adds dns_servers option for specifying DNS servers for name resolution. Accepts hostnames or IP addresses in the same format as the except option.
```

```suggestion
  - nmap inventory plugin - adds ``dns_servers`` option for specifying DNS servers for name resolution. Accepts hostnames or IP addresses in the same format as the ``except`` option (https://github.com/ansible-collections/community.general/pull/9849).
```
Could it be that you mean `exclude` and not `except`?